apps/src/templates/teacherHomepage/TeacherHomepage.story.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import TeacherHomepage from './TeacherHomepage';
+
+export default storybook => {
+  return storybook
+    .storiesOf('TeacherHomepage', module)
+    .addStoryTable([
+      {
+        name: 'Teacher Homepage',
+        description: 'Teacher Homepage - teacher has course progress, but does not have sections',
+        story: () => (
+          <TeacherHomepage
+            announcements={[{
+              heading: "Go beyond an Hour of Code",
+              buttonText: "Go Beyond",
+              description: "Go Beyond an Hour of Code and explore computer science concepts with your students every week. Code.org offers curriculum, lesson plans, high quality professional learning programs, and tons of great tools for all grade levels - and it's free. No experience required - find the next step that's right for your classroom.",
+              link: "to wherever"
+            }]}
+            courses= {[
+              { courseName: "Play Lab",
```

Comment:
nit: here and elsewhere, `courseName` should be on a new line following opening brace.